client/packages/electron/src/electron.ts

```diff
@@ -225,3 +202,55 @@ process.on('uncaughtException', error => {
         at process.processTimers (node:internal/timers:502:7)
   */
 });
+
+// App data store
+type StoreType = {
+  [key: string]: string | null;
+};
+const store = new ElectronStore<StoreType>();
+// /Users/andreievg/Library/Application Support/open mSupply
+// make CLI for generating certs
+app.addListener(
+  'certificate-error',
+  (event, _webContents, url, _error, certificate, callback) => {
+    // Ignore SSL checks in debug mode
+    if (getDebugHost()) {
+      event.preventDefault();
+      return callback(true);
+    }
+
+    // Default behaviour if not connected to a server or if url is not connectedServer
+
+    if (!connectedServer) return callback(false);
+
+    if (!url.startsWith(frontEndHostUrl(connectedServer))) {
+      return callback(false);
+    }
+
+    // Match SSL fingerprint for server stored in app data
+
+    // Match by hardware id and port
+    let identifier = `${connectedServer.hardwareId}-${connectedServer.port}`;
+    let storedFingerprint = store.get(identifier, null);
+
+    // If fingerprint does not exists for server add it
+    if (!storedFingerprint) {
+      storedFingerprint = certificate.fingerprint;
```

Comment:
`certificate.fingerprint` is build in into electron isn't it? or do we set it?

Reply:
this one is set by electron